app/services/spine_service/reports/clinic/attendance_report.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module SpineService
+  module Reports
+    module Clinic
+      class AttendanceReport
+        include ModelUtils
+        attr_reader :start_date, :end_date
+
+        def initialize(start_date:, end_date:, **kwargs)
+          @start_date = start_date.to_date.beginning_of_day.strftime('%Y-%m-%d %H:%M:%S')
+          @end_date = end_date.to_date.end_of_day.strftime('%Y-%m-%d %H:%M:%S')
+        end
+
+        def fetch_report
+          data = Encounter.where(
+              'encounter_datetime BETWEEN ? AND ? AND program_id = ? AND voided = 0', 
```

Comment:
There is no need for voided = 0 because for all Encounters they have a default scope the appends ```where voided = 0```